src/modules/auth/header.js

```diff
@@ -11,12 +11,15 @@ import {
 function HeaderComponent(props) {
 
 	const Config = props.app.Config;
```

Comment:
You want to do:

```
let user_auth_url = props.app.getServiceURL('auth_webui')
```

... and only if this fails, you handle backward compatibility.
Always try to improve the code when you visit it, it pays of greatly.

Reply:
@ateska Well, this will not fail when `auth_webui` will not be defined in SERVICES. If `auth_webui` is not defined in SERVICES, then the `user_auth_url` by your proposal will look like this

```
user_auth_url = 'https://adminpanel.url.com/api/auth_webui'
```

And thus it will not return the desired output of `undefined` and proceed with backward compatibility, because in Application.js it is set like this to allow all the API endpoints (not only those mentioned in SERVICES) and create a service URL from it:

https://github.com/TeskaLabs/asab-webui/blob/093f490ea76cd48f66eba2892532b4c06917dc50/src/containers/Application.js#L208

So if we want to check, if the `auth_webui` is defined/undefined, we need to check it in the configuration and then pass the result to the `getServiceURL(Config.SERVICES.auth_webui)` as it is in proposal in the last commit. https://github.com/TeskaLabs/asab-webui/pull/78/commits/7e49a0d89693a647dde117e0f450f1ca222af9ef

In here, when authPath is undefined, https://github.com/TeskaLabs/asab-webui/blob/7e49a0d89693a647dde117e0f450f1ca222af9ef/src/modules/auth/header.js#L16 , the `user_auth_url` will be also undefined https://github.com/TeskaLabs/asab-webui/blob/7e49a0d89693a647dde117e0f450f1ca222af9ef/src/modules/auth/header.js#L17 and then we can proceed with the backward compatibility

Or then we want to check specifically for `auth_webui` in `getServiceURL` as we do it for `oidc` and `rbac`.